addons/point_of_sale/models/pos_session.py

```diff
@@ -101,6 +101,274 @@ class PosSession(models.Model):
 
     _sql_constraints = [('uniq_name', 'unique(name)', "The name of this POS Session must be unique!")]
 
+    # We need to pass config_id because sometime params are needed without opened session (eg: closed self order).
+    # If we don't have a session, we can't get the config_id from the session.
+    def _load_data_params(self, config_id):
+        params = {
+            'search_read': {
+                'pos.config': {
+                    'domain': [('id', '=', config_id.id)],
+                    'fields': []
+                },
+                'pos.order': {
+                    'fields': [],
+                    'domain': [],
+                },
+                'pos.order.line': {
+                    'domain': lambda data: [('order_id', 'in', [order['id'] for order in data['pos.order']])],
+                    'fields': [
+                        'qty', 'attribute_value_ids', 'custom_attribute_value_ids', 'price_unit', 'skip_change', 'uuid', 'price_subtotal', 'price_subtotal_incl', 'order_id',
+                        'product_id', 'discount', 'tax_ids', 'pack_lot_ids', 'customer_note', 'refunded_qty', 'price_extra', 'full_product_name', 'refunded_orderline_id', 'combo_parent_id', 'combo_line_ids'],
+                },
```

Comment:
There seems to be no need to load the `pos.order` and `pos.order.line` this way. Most of the use cases I saw in this PR is mainly from 'data', not really from 'models'. But maybe I'm missing.

Reply:
I will comment this,